tests/test_extension.py

```diff
@@ -92,9 +92,9 @@ def test_serve_swagger_ui_custom_url(self, app, client):
     def test_apispec_config(self, app):
         app.config['APISPEC_TITLE'] = 'test-extension'
         app.config['APISPEC_VERSION'] = '2.1'
+        app.config['APISPEC_OA_VERSION'] = '2.0'
```

Comment:
How about `APISPEC_OAS_VERSION`, for consistency with the [official abbreviation](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#introduction)?

Reply:
Agreed, best to stick with the offical abbrev., fixed.